editor/sidebar/post-author/index.js

```diff
@@ -73,15 +74,23 @@ class PostAuthor extends Component {
 	}
 }
 
-export default connect(
-	( state ) => {
+export default flowRight( [
+	connect(
+		( state ) => {
+			return {
+				postAuthor: getEditedPostAttribute( state, 'author' ),
+			};
+		},
+		{
+			onUpdateAuthor( author ) {
+				return editPost( { author } );
+			},
+		},
+	),
+	withAPIData( () => {
 		return {
-			postAuthor: getEditedPostAttribute( state, 'author' ),
+			users: '/wp/v2/users?context=edit&per_page=100',
 		};
-	},
-	{
-		onUpdateAuthor( author ) {
-			return editPost( { author } );
-		},
-	},
-)( withInstanceId( PostAuthor ) );
+	} ),
```

Comment:
Maybe. Naming things is hard :smile: For example, the API data higher-order component doesn't necessarily incur a fetch (might be used only for updating), and itself doesn't perform the fetch. All it does is apply a wrapping (higher-order) component.

I think if we were to, we'd want some easy convention for naming, like a consistent prefix. Maybe "apply": `applyConnect`, `applyWithAPIData`